### DISCUSSION TITLE
Developer Meeting: Generic support (Jan 3 2024)

### DISCUSSION THREAD

**Discussion by Bananazooka** [bitcraze/discussions#1099] [Announcements]
The next developer meeting will be on the 3rd of January. It will a general support meeting, where you will have the opportunity to ask questions to @knmcguire.

Date: Wednesday, Jan 3, 2024
Time: 3 PM Central European Time
Topic: Support
Hosts: @knmcguire
Meeting link: _Meeting has ended, no recording available_

Indicate with 👍 if you are planning to join!

**Comment by knmcguire** [comment]
Just a reminder for anyone that wants to have 1-1 support: tomorrow's devmeeting will not be about any topic but just to handle questions.

**Comment by knmcguire** [comment]
Thanks for joining! 

I have some followup info for some of the questions:
- The issue of the crazyflie not keeping a larger distance from the wall, I've made an issue for that here: https://github.com/bitcraze/crazyflie-lib-python/issues/435
- How to sniff CRTP packets in air. This is not possible with a Crazyradio PA but it is with a BBC microbit or to turn one of the crazyflies into a sniffer:
   - microbit board: https://forum.bitcraze.io/viewtopic.php?p=19111&hilit=microbit#p19111
   - Crazyflie sniffer as part of the decentralized swarm example: https://github.com/bitcraze/crazyflie-firmware-experimental/blob/icra-2023/examples/demos/decentralized_swarm/src/sniffer.c
- Control the crazyflies as a swarm:
   - Swarm examples https://github.com/bitcraze/crazyflie-lib-python/tree/master/examples/swarm
   - step by step guide for beginners: https://www.bitcraze.io/documentation/repository/crazyflie-lib-python/master/user-guides/sbs_swarm_interface/ 
   - Crazyswarm2 project to control with ROS2: https://github.com/IMRCLab/crazyswarm2
 - About our roadmap with the crazyradio that is in the pipeline:
   -  Blogpost by arnaud: https://www.bitcraze.io/2023/01/crazyradio-2-0/ 
   - Presentation by arnaud at the microros group: https://youtu.be/yJZ7TiPK7gU?feature=shared
   - Perhaps an alternative way of controlling the crazyflie (other than bluetooth) is wifi: https://www.bitcraze.io/2022/09/using-the-ai-deck-for-crtp-over-wifi-via-cpx/. Currently we only have the aideck with wifi but we are thinking about prototypes of a wifi only deck that might be a good option.
